src/js/components/Carousel/__tests__/__snapshots__/Carousel-test.js.snap

```diff
@@ -121,8 +121,8 @@ exports[`Carousel basic 1`] = `
   -ms-flex-direction: column;
   flex-direction: column;
   opacity: 1;
-  -webkit-animation: hftlBD 1s 0s forwards;
-  animation: hftlBD 1s 0s forwards;
+  -webkit-animation: hftlBD 0s 0s forwards;
```

Comment:
@halocline had been exploring this snapshot changes with the following insights:
>The snapshot changes for Carousel now represent the intended logic.
L158 specifies an animation duration of ‘0’, which is passed to Box’s animation on L164.
The value of 0 is evaluated (incorrectly) in the normalizeTiming function here. Instead of the function interpreting the value as 0 , it is evaluating it as a boolean and therefore interprets 0 as falsey. As consequence, normalizeTiming() had been returning the defaultTiming of 1s instead of the intended 0s.
Jonathan’s PR modifies the normalizeTiming() function and correctly interprets 0 as a numeric value and correctly returns 0s. This is the source of Carousel’s snapshot changes.

Reply:
Yup! This is exactly what @halocline and I discussed!